components/DataSchema.tsx

```diff
@@ -106,10 +90,86 @@ function getColumnDef(dataSchemaMap?: {
             selector: ColumnSelector.Required,
             wrap: true,
             sortable: true,
-            // TODO it may not be accurate to use the `required` field because an attribute may be listed as a
-            //  required dependency for the parent attribute even if `required` field is false
             format: (schemaData: DataSchemaData) =>
-                schemaData.required ? 'Yes' : 'No',
+                schemaData.required ? 'True' : 'False',
+        },
+        [ColumnName.ConditionalIf]: {
+            name: ColumnName.ConditionalIf,
+            selector: ColumnSelector.ConditionalIf,
+            cell: (schemaData: DataSchemaData) => {
+                const conditionalIfList = [];
+
+                // Check if this schema is a dependency of any other schema
+                if (dataSchemaMap) {
+                    for (const [key, value] of Object.entries(dataSchemaMap)) {
+                        if (
+                            value.requiredDependencies &&
+                            Array.isArray(value.requiredDependencies)
+                        ) {
+                            const isDependency = value.requiredDependencies.some(
+                                (dep) =>
+                                    (typeof dep === 'string'
+                                        ? dep
+                                        : dep['@id']) === schemaData.id
+                            );
+                            if (isDependency && value.attribute) {
+                                conditionalIfList.push(value.attribute);
+                            }
+                        }
+                    }
+                }
```

Comment:
Better to have this logic as a function in `dataSchemaHelpers`. Something like `findConditionalAttributes`.